conf/surveys/HST.yaml

```diff
@@ -1,8 +1,10 @@
-# Source https://hst-docs.stsci.edu/display/WFC3IHB/6.6+UVIS+Optical+Performance#id-6
+# pixel_scale: https://hst-docs.stsci.edu/acsihb/chapter-3-acs-capabilities-design-and-operations/3-5-acs-quick-reference-guide
+# exp_time: https://hst-docs.stsci.edu/acsdhb/chapter-1-acs-overview/1-2-basic-instrument-operations
+# fwhm: https://hst-docs.stsci.edu/display/WFC3IHB/6.6+UVIS+Optical+Performance#id-6
 # .6UVISOpticalPerformance-6.6.1 800nm
 HST:
   name: "HST"
-  pixel_scale: 0.06
+  pixel_scale: 0.05
```

Comment:
I think here we should use the parameters for COSMOS specifically (maybe we should rename the survey as HST_COSMOS ?). In that case, the pixel scale is actually 0.03 (they used multidrizzle), and the parameters are from the ACS as you surmised.

Reply:
Yes this is also what I thought about. If @ismael-mendoza is ok I can change HST to HST_COSMOS and set the corresponding parameters.

Reply:
yes please go ahead! @mpaillassa